src/main/resources/anniv5Resources/localization/eng/Cardstrings.json

```diff
@@ -9,7 +9,7 @@
   },
   "${ModID}:Rummage": {
     "NAME": "Rummage",
-    "DESCRIPTION": "{@@}Deal !D! damage. NL Draw a card and reduce its cost by !M! this turn."
+    "DESCRIPTION": "{@@}Deal !D! damage. NL Draw !${ModID}:m2! {@@}{!${ModID}:m2!|@=card|>1=cards} and reduce {@@}{!${ModID}:m2!|@=its|>1=their} cost by !M! this turn."
```

Comment:
Don't forget to overwrite the zhs strings so mwalls can easily see what to change

Reply:
what do you mean with overwite the zhs string?